### PR TITLE
More accessibility improvements

### PR DESCRIPTION
**Pull request by andygnewman**
Knocking more items off this list in issue https://github.com/Financial-Times/next-article/issues/549

Specifically;

Main category link needs aria-label with appropriate description
Topics mentioned link list should be an aside w/ role="complementary"
Promoboxes / Story packages to other articles should be an aside w/ role="complementary" 
More on/Related topic areas should be each have role="region'
<abbr> elements require text content (but hidden to non-screenreaders)

### CODE REVIEW

**Comment by ironsidevsquincy** [comment]
looks good :+1:

File: views/article-v1.html [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 			<div data-o-grid-colspan="12 L8 XL7 XLoffset1">
-				<h3 class="article__title">{{{title}}}</h3>
-				{{#if standFirst}}
-					<h4 class="article__stand-first">{{{standFirst}}}</h4>
-				{{/if}}
+				<header>
+					<h1 class="article__title">{{{title}}}</h1>
+					{{#if standFirst}}
+					<p> class="article__stand-first">{{{standFirst}}}</p>
+					{{/if}}
```
Comment (i-like-robots) [inline views/article-v1.html]: Oops

File: views/article-v1.html [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 		<div class="article__header-inner o-grid-row">
 			<div data-o-grid-colspan="12 L8 XL7 XLoffset1">
-				<h3 class="article__title">{{{title}}}</h3>
-				{{#if standFirst}}
-					<h4 class="article__stand-first">{{{standFirst}}}</h4>
-				{{/if}}
+				<header>
+					<h1 class="article__title">{{{title}}}</h1>
+					{{#if standFirst}}
+					<p> class="article__stand-first">{{{standFirst}}}</p>
```
Comment (i-like-robots) [inline views/article-v1.html]: Oops

File: views/partials/reading-list.html [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <div class="myft-reading-list js-myft-reading-list" data-trackable="myft-reading-list" data-o-grid-colspan="12 L8 XL7 XLoffset1" data-o-component="o-expander">
 	<a class="myft-reading-list__title" href="/myft/my-news" data-trackable="continue">
-		<span class="myft-reading-list__title-text">Continue reading your personal news feed at<abbr class="myft-ui__icon" title="myFT"></abbr></span>
+		<span class="myft-reading-list__title-text">Continue reading your personal news feed at<abbr class="myft-ui__icon" title="myFT"><span class="utility__text-hidden">My FT</span></abbr></span>
```
Comment (i-like-robots) [inline views/partials/reading-list.html]: This shouldn't be an `<abbr>` at all, just a span (no title req.)

File: views/partials/reading-list.html [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 	</a>
 	<div class="more-on__content">
 
 		<div class="myft-reading-list__footer">
-			<a href="/myft/my-news" data-trackable="myft-icon"><abbr class="myft-reading-list__red-icon" title="myFT"></abbr></a>
+			<a href="/myft/my-news" data-trackable="myft-icon"><abbr class="myft-reading-list__red-icon" title="myFT"><span class="utility__text-hidden">My FT</span></abbr></a>
```
Comment (i-like-robots) [inline views/partials/reading-list.html]: As above

File: views/article-v2.html [hunk 5]
```diff
@@ -39,7 +44,7 @@ <h1 class="article__title">{{{title}}}</h1>
 					<div class="o-grid-row">
 						{{#if primaryTag}}
 						<div data-o-grid-colspan="12 S6" class="article__primary-theme">
-						<a href="/stream/{{primaryTag.taxonomy}}Id/{{primaryTag.id}}" class="article__primary-theme__link ng-title-link" data-trackable="section-link">{{primaryTag.name}}</a>
+						<a href="/stream/{{primaryTag.taxonomy}}Id/{{primaryTag.id}}" class="article__primary-theme__link ng-title-link" data-trackable="section-link" aria-label="in-category-{{primaryTag.name}}">{{primaryTag.name}}</a>
```
Comment (i-like-robots) [inline views/article-v2.html]: The `aria-label` can doesn't require formatting, "posted in the category bla bla bla" is fine, no need to hyphen separate.

**Comment by i-like-robots** [comment]
Looks good, just one comment. Rebase then :ship:

**Comment by i-like-robots** [comment]
:ship:

**Pull request merged**